MMA.md

```diff
@@ -0,0 +1,212 @@
+
+Events:
+Most of the Leanplum events can be mapped to a single combination of Telemetry event (Event+Method+Extra).
+Some events are not collected in Mozilla Telemetry. This will be addressed separately in each campaign review.
+There are three elements that are used for each event. They are: event name, value(default: 0.0), and info(default: "").
+Default value for event value is 0.0. Default value for event info is empty string.
+
+Here is the list of current Events sent, which can be found here in the code base: https://github.com/mozilla-mobile/firefox-ios/blob/master/Client/Application/LeanplumIntegration.swift#L21
+
+The first launch after install
+~~~~
+{
+  "event": "E_First_Run"
+}
+~~~~
+
+The second launch after install
+~~~~
+{
+  "event": "E_Second_Run"
+}
+~~~~
+
+Whenever the App is launched.
+~~~~
+{
+  "event": "E_Opened_App"
+}
+~~~~
+
+The user loads a bookmark from home panel.
+~~~~
+{
+  "event": "E_Opened_Bookmark"
+}
+~~~~
+
+The user opened a new tab.
+~~~~
+{
+
+  "event": "E_Opened_New_Tab"
+}
+~~~~
+
+The user interacts with search url area.
+~~~~
+{
+  "event": "E_Interact_With_Search_URL_Area"
+}
+~~~~
+
+The user saves a bookmark.
+~~~~
+{
+  "event": "E_Saved_Bookmark"
+}
+~~~~
+
+The user opened a mailto link from a web page.
+~~~~
+{
+  "event": "E_Opened_Mailto_Link"
+}
+~~~~
+
+The user cleared their private data.
+~~~~
+{
+  "event": "E_Cleared_Private_Data"
+}
+~~~~
+
+Download videos or any other media
+~~~~
+{
+  "event": "E_Download_Media_Saved_Image"
+}
+~~~~
+
+Save password and login from door hanger
+~~~~
+{
+  "event": "E_Saved_Login_And_Password"
+}
+~~~~
+
+Deep Links:
+Deep links are actions that can point Firefox to open certain pages or load features such as `show bookmark list` or
+`open Firefox Account Settings`. When users receives a Message, they can click the button(s) on it. These buttons can
+trigger the following deep links:
+* Link to sync signup/sign in (firefox://deep-link?url=settings/fxa)
+* Link to default search engine settings (firefox://deep-link?url=settings/search)
+* Link to bookmark list (firefox://deep-link?url=homepanel/bookmarks)
+* Link to history list (firefox://deep-link?url=/history)
+* Link to general preferences (firefox://deep-link?url=settings)
+* Link to the mailto preferences (firefox://deep-link?url=settings/mailto)
+* Link to open a new tab (firefox://deep-link?url=settings/newtab)
+* Link to open the user's homepage (firefox://deep-link?url=settings/homepage)
+* Link to open the settings page so the user can clear their private data (firefox://deep-link?url=settings/clear-private-data)
+* Link to open a new private tab (firefox://deep-link?url=homepanel/new-private-tab)
+
+
+Note these deep links can only be processed from within Firefox (i.e., if they are clicked from outside of Firefox, Firefox will open but nothing further will happen).
+
+Messages :
```

Comment:
One nit: I'd make this a header! Messages are an important section, so let's give it some kind of section header (doesn't need to be top level, whatever level seems appropriate).